docs/user-guide/release-notes/2.13.x.rst

```diff
@@ -11,10 +11,15 @@ New Features
 * Log statements which are emitted from task processes now include a short task
   id of 8 characters (i.e. ``pulp.server.sync::INFO: [0ca4399c] ...``) which
   can be used to look up a task.
+
 * Users no longer need to have pulp_celerybeat running to know whether pulp_workers or
   pulp_resource_manager are online. All workers now write their own heartbeat records to the
   database, eliminating the dependence on pulp_celerybeat.
 
+* serial_number_path removed from server.conf. Pulp now generates the serial number
+  randomly and does not use the sn.dat file. This is a part of the `bugfix for #1839
```

Comment:
I think using the macros we have would be an improvement. The second for [here](http://docs.pulpproject.org/dev-guide/contributing/documenting.html#editing-the-docs) is equivalent to what you've written.